api/server.ts

```diff
@@ -10,3 +11,4 @@ app.post('/', (req, res) => {
 });
```

Comment:
This is unnecessary but maybe instead of sending 'hello world', send a short message like 'connected' or something.

Reply:
This section is just a placeholder for the root / base endpoint, which is unrelated to my issue.